pages/common/pueue-stash.md

```diff
@@ -0,0 +1,22 @@
+# pueue stash
+
+> Stash tasks to prevent automatic start.
+> See also: `pueue start`.
+> See also: `pueue enqueue`.
```

Comment:
Maintainers: is it better to condense these? (https://github.com/tldr-pages/tldr/pull/5285#issuecomment-784167699)?

Reply:
Yes. I've left a comment in my review.